test/evergreen.yml

```diff
@@ -1783,6 +1806,33 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    #set a 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger sanitizer"
+      - func: "format test script"
+        vars:
+          test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
+          # run for 24 hours ( 24 * 60 = 1440 minutes), stop at first error, use default config
+          format_test_script_args: -F -t 60
```

Comment:
Assuming the `-t` setting will be adjusted to 24hrs. 
We may not want the task stops upon the first failure, instead tolerate the failures and run through 24hrs.

Reply:
So essentially remove the -F tag and let run the script running for full cycle?

Reply:
Yup, running full cycle of 24hrs is to simulate continuous testing we currently have in Jenkins.